tests/settings.test.ts

```diff
@@ -146,6 +149,41 @@ describe.each([{ permission: 'Master' }, { permission: 'Admin' }])(
       expect(response).toMatchSnapshot()
     })
 
+    test(`${permission} key: Update embedders settings `, async () => {
```

Comment:
Creation of a new test to test the embedder's addition on the `settings` route (it was forgotten previously). 
Since it needs the activation of the `vectorStore` exp. feature, I thought it would be best to create a specific test for this one